### PR TITLE
Automated acceptance tests for EKS

### PR DESCRIPTION
**Pull request by irinaepshteyn**

### CODE REVIEW

**Review by Yogeshkumarb57 — APPROVED** [review]

File: AcceptanceEKSJenkinsfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                }
+                sh '''#!/bin/bash -ex
+                    echo 'Installing yq'
+                    wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64
+                    chmod a+x /usr/local/bin/yq
+                    yq --version
+
+                    source uaa/scripts/setup-tests.sh
+
+                    export JRE_VERSION="{ jre: { version: 11.+ }}"
```
Comment (akarollil) [inline AcceptanceEKSJenkinsfile:64]: Do we need these lines?

File: AcceptanceEKSJenkinsfile [hunk 2]
```diff
@@ -58,6 +62,10 @@ pipeline {
                     source uaa/scripts/setup-tests.sh
 
                     export JRE_VERSION="{ jre: { version: 11.+ }}"
+                    export ARTIFACTORY_READER=$FUNCTIONAL_USER_USR
+                    export ARTIFACTORY_READER_PW=$FUNCTIONAL_USER_PSW
+                    export SAML_IDP_USER=$FUNCTIONAL_USER_USR
+                    export SAML_IDP_USER_PW=$FUNCTIONAL_USER_PSW
```
Comment (akarollil) [inline AcceptanceEKSJenkinsfile:68]: So the SAML IDP credentials are the same as those for Artifactory reader?

File: AcceptanceEKSJenkinsfile (hunk 1 above)
Comment (irinaepshteyn) [reply AcceptanceEKSJenkinsfile:64]: Good catch. It's needed by the CF deploy script, but not needed for running acceptance tests.  JRE environment here is set by Jenkins agent that is build on top of gradle:6.3.0-jdk11

File: AcceptanceEKSJenkinsfile (hunk 2 above)
Comment (irinaepshteyn) [reply AcceptanceEKSJenkinsfile:68]: Yes, we are using the same GE SSO functional user

File: AcceptanceEKSJenkinsfile (hunk 1 above)
Comment (irinaepshteyn) [reply AcceptanceEKSJenkinsfile]: Removed JRE_VERSION

Comment (akarollil) [reply AcceptanceEKSJenkinsfile]: What about setup-tests.sh? It seems to be just unsetting things? Probably not needed?

Comment (irinaepshteyn) [reply AcceptanceEKSJenkinsfile]: Oops... not needed either.  Removed

**Review by akarollil — APPROVED** [review]

**Pull request merged**